packages/vite/src/node/server/pluginContainer.ts

```diff
@@ -683,13 +690,16 @@ export async function createPluginContainer(
           if (isObject(result)) {
             updateModuleInfo(id, result)
           }
+          perf.collect(`plugin::load::${relativeUrl(id, root)}`, start)
```

Comment:
🤔 Although `perf.collect` will return early if `perf` is not enabled, seems like this still result and perf regression as it might be invoked by too many times.
Here's the benchmark https://github.com/fi3ework/vite-benchmark/actions/runs/4644839555, I run triple times, and all shows 1000 React components case will got a perf regression, probably because it is more obvious when there are vast components.

Reply:
Ah that may lead too many extra `perf.collect` calls.  The plugin API timing collector may move to `vite-plugin-inspect` later.